paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsAdapter.kt

```diff
@@ -593,131 +599,129 @@ internal fun PaymentOptionUi(
 ) {
     // An attempt was made to not use constraint layout here but it was unsuccessful in
     // precisely positioning the check and delete icons to match the mocks.
-    PaymentsTheme {
-        ConstraintLayout(
+    ConstraintLayout(
```

Comment:
This looks like I changed way more than I did. I just removed the `PaymentsTheme` being set.